<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainFormClass</class>
 <widget class="QMainWindow" name="MainFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1272</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainForm</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="ecuFileLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>No ECU file loaded</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="descriptorFileLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>No DescriptorPack loaded</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="descriptorSearchLineEdit">
          <property name="placeholderText">
           <string>Search descriptor names</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="mapsTreeView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QMdiArea" name="mainArea">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="viewMode">
        <enum>QMdiArea::SubWindowView</enum>
       </property>
       <property name="documentMode">
        <bool>false</bool>
       </property>
       <property name="tabsClosable">
        <bool>false</bool>
       </property>
       <property name="tabsMovable">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1272</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_project"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_ECU_File"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_DescriptorPack"/>
    <addaction name="actionConvert_Csv_to_DescriptorPack"/>
    <addaction name="actionSave_DescriptorPack"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_ECU_File">
   <property name="text">
    <string>Load ECU File</string>
   </property>
  </action>
  <action name="actionLoad_DescriptorPack">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load DescriptorPack</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionConvert_Csv_to_DescriptorPack">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Convert Csv to DescriptorPack</string>
   </property>
  </action>
  <action name="actionNew_project">
   <property name="text">
    <string>New project</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionSave_DescriptorPack">
   <property name="text">
    <string>Save DescriptorPack</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainForm.qrc"/>
 </resources>
 <connections/>
</ui>
